cmd/podman/common/create_opts.go

```diff
@@ -202,14 +202,14 @@ func ContainerCreateToContainerCLIOpts(cc handlers.CreateContainerConfig, cgroup
 	// TODO options
 	mounts := make([]string, 0, len(cc.HostConfig.Mounts))
 	for _, m := range cc.HostConfig.Mounts {
-		mount := fmt.Sprintf("type=%s", m.Type)
+		mount := fmt.Sprintf("type=%s,", m.Type)
 		if len(m.Source) > 0 {
-			mount += fmt.Sprintf("source=%s", m.Source)
+			mount += fmt.Sprintf("source=%s,", m.Source)
```

Comment:
Wouldn't it be simpler to prepend the comma here, and remove the `TrimSuffix()` below?
```go
    mount += fmt.Sprintf(",source=%s", m.Source)
                          ^
```